apps/Steam/install-64

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+install_packages libnss3:armhf libnm0:armhf libdbus-glib-1-2:armhf libudev1:armhf libnspr4:armhf libgudev-1.0-0:armhf libusb-1.0-0:armhf libappindicator1:armhf libnss3 libnm0 libdbus-glib-1-2 libudev1 libnspr4 libgudev-1.0-0 libusb-1.0-0 libappindicator1 || exit 1
+
+if package_installed steam-devices ; then
+  echo "Removing steam-devices package because it conflicts with steam-launcher..."
+  sudo apt purge -y steam-devices
+fi
+
+if package_installed steamlink ; then
+  echo "Removing steamlink package because it conflicts with steam-launcher..."
+  sudo apt purge -y steamlink
+fi
+
+"${DIRECTORY}/manage" install-if-not-installed Box86 || error "Box86 failed to install somehow!"
+"${DIRECTORY}/manage" install-if-not-installed Box64 || error "Box64 failed to install somehow!"
+
+echo "Downloading steam.deb"
+wget https://steamcdn-a.akamaihd.net/client/installer/steam.deb || error "Failed to download steam.deb!"
+
+sudo dpkg -i ~/steam.deb || error "dpkg failed to install steam.deb!"
+
+rm ~/steam.deb
+
+echo "Editing steam script to run on Raspberry Pi..."
+sudo sed -i 's/set -e/set -e\nexport STEAMOS=1\nexport STEAM_RUNTIME=1\nexport BOX86_NOGTK=1/' /usr/lib/steam/bin_steam.sh
+
+# echo "Creating a Steam config file so it will launch in Small Mode..."
+# mkdir -p ~/.local/share/Steam/config
+# wget https://raw.githubusercontent.com/Botspot/Steam-RPi/main/DialogConfig.vdf -O ~/.local/share/Steam/config/DialogConfig.vdf
```

Comment:
Why are these lines commented out? Does Steam launch fine in Normal Mode on 64-bit?

Reply:
in theory it should.... steam's normal mode uses a x86_64 chrome-sandbox, obviously that can't be emulated on arm32 with box86, but if box64 should be able to do it on arm64. however, the readme for box64 lists steam as not compatible so your milage may vary

Reply:
https://discord.com/channels/779861777386569768/847179308539183134/949727209772437544

As per the above message it should work but not tested

Reply:
It launches in full screen by default, but still nothing shows up it's just a black window. 
![image](https://user-images.githubusercontent.com/71036629/160210547-e01fa392-8c79-460f-b4a8-45f5bd42cfcd.png)

Small mode still works though. I think these lines should be added back.